core/client/app/controllers/settings/general.js

```diff
@@ -76,6 +76,20 @@ export default Controller.extend(SettingsSaveMixin, {
         }
     }),
 
+    facebookUrl: computed('model.facebook', function () {
+        return this.get('model.facebook');
+    }),
+
+    twitterUrl: computed('model.twitter', function () {
+        let url = 'http://twitter.com/';
+        let modelVal = this.get('model.twitter');
+        let [ , user ] = modelVal.match(/@?([^\/]*)/);
```

Comment:
I think this is where the acceptance tests are failing, you'll need to either update this piece of code to account for `model.twitter` being missing and update the test settings fixtures in `core/client/app/mirage/fixtures/settings.js` if there are any tests which expect the facebook/twitter data to be present.